modules/nlu/src/backend/pipelines/intents/svm_classifier.ts

```diff
@@ -304,9 +326,18 @@ export default class SVMClassifier {
       token2vec: this.token2vec
     })
 
+    const input = tokens.join(' ')
```

Comment:
why do we join tokens with whitespaces ? don't we now have the __ char for this ?

Reply:
Only for logs